clients/src/test/java/org/apache/kafka/common/utils/ImplicitLinkedHashMultiCollectionTest.java

```diff
@@ -114,11 +114,11 @@ public void testEnlargement() {
             new TestElement(101),
             new TestElement(105)
         };
-        for (int i = 0; i < testElements.length; i++) {
-            assertTrue(multiSet.add(testElements[i]));
+        for (TestElement testElement : testElements) {
+            assertTrue(multiSet.add(testElement));
         }
-        for (int i = 0; i < testElements.length; i++) {
-            assertFalse(multiSet.add(testElements[i]));
+        for (TestElement testElement : testElements) {
+            assertFalse(multiSet.add(testElement));
```

Comment:
could this be simplified to a single loop with the 2 asserts in it? we would achieve the same, validate an elem can be added only if new, for all elems.

Reply:
Agreed, we can merge both loops